egs2/ljspeech/tts1/local/data.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+
+set -e
+set -u
+set -o pipefail
+
+
+log() {
+  local fname=${BASH_SOURCE[1]##*/}
+  echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+  }
+SECONDS=0
+
+stage=1
+stop_stage=100
+
+train_set=
+dev_set=
+eval_set=
+trans_type=
+
+datasets_root=/home/shaunxliu/data/datasets
+
```

Comment:
Refer ${LJSPEECH} in db.sh

See the following for example: 
https://github.com/espnet/espnet/blob/23d06901f388888f7608979ba3b1a287d749c4f0/egs2/jsut/asr1/local/data.sh#L25-L38